packages/botonic-core/src/core-bot.js

```diff
@@ -54,7 +56,16 @@ export class CoreBot {
     if (!session.__locale) session.__locale = 'en'
 
     if (this.plugins) {
-      await runPlugins(this.plugins, 'pre', input, session, lastRoutePath)
+      await runPlugins(
+        this.plugins,
+        'pre',
+        input,
+        session,
+        lastRoutePath,
+        undefined,
+        undefined,
+        this.dataProvider
+      )
```

Comment:
Maybe it would be better to use named parameters using an object in order to avoid initializing these 2 `undefined` and to know what each argument refers to.

Reply:
added a task for this, we have to be sure we don't break current functionality: https://linear.app/hubtype/issue/BOT-541/simplify-calls-to-loadrun-plugins-in-botoniccore